splunk_connect_for_snmp_poller/manager/validator/inventory_validator.py

```diff
@@ -123,7 +123,7 @@ def is_valid_inventory_line_from_dict(host, version, community, profile, seconds
         and is_valid_version(version.strip())
         and is_valid_community(community.strip())
         and is_valid_profile(profile.strip())
-        and is_valid_second_quantity(seconds.strip())
+        and is_valid_second_quantity(seconds)
```

Comment:
are we performing strip earlier? or are we sure that frequency from profile will not need strip?

Reply:
Strip was required if frequency was read from csv files. Currently it is stored in yaml file as "frequency" : 20 and is interpreted as int not string. It is read correctly even if spaces are present before or after the number.